src/usr/local/www/firewall_rules_edit.php

```diff
@@ -296,6 +296,7 @@ function is_aoadv_used($rule_config) {
 		$pconfig['interface'] = $_GET['if'];
 	}
 	$pconfig['type'] = "pass";
+	$pconfig['proto'] = "tcp"; // ensures port fields visible and default=tcp, for a new rule
```

Comment:
It seems to work but it's still a change unrelated to this PR. That belongs in a separate PR.